dawn/renderer/components/FinalCompPeripheralList.js

```diff
@@ -22,11 +22,11 @@ const FinalCompPeripheralList = (props) => {
         _.toArray(props.peripherals),
         (peripheral) => (
           <Peripheral
-            key={peripheral.id}
-            id={peripheral.id}
-            name={peripheral.name}
+            key={peripheral.uid.low}
```

Comment:
Throwing away the upper bits of the device UID seems extremely suspect.

If the UID is ever displayed in the UI, it will be truncated and therefore not the same thing that staff would see on any debug tools.

If it's ever used for non-display purposes (e.g. name-to-uid mapping), then throwing away those bits may have even worse consequences in terms of making the mapping unusable for runtime/hibike.